### ISSUE TITLE
GDB tests fail on Jenkins

### ISSUE THREAD

**Issue by pwmarcz** [oscarlab/graphene#1871]
See https://github.com/oscarlab/graphene/pull/1868.

GDB tests (`in test_libos.py`, currently skipped) fail on some slaves, as far as I can tell, the program inside GDB fails to start:

```
Standard Output

[graphene_gdb.py] Loading /home/jenkins/workspace/graphene-debug-18.04/Runtime/../Pal/src/host/Linux/gdb_integration/graphene.gdb...
Breakpoint 1 at 0x19120: file pal_start-x86_64.S, line 13.
Breakpoint 2 at 0x19180: file clone-x86_64.S, line 86.
Catchpoint 3 (vfork)
Temporary breakpoint 4 (func) pending.

Standard Error

Function "func" not defined.
debug.gdb:26: Error in sourced command file:
During startup program exited with code 127.
```

* The program fails to start before executing anything (even `starti` fails)
* The test fails on some Jenkins nodes but not all of them; on both Ubuntu 16.04 and 18.04 Docker images
  * Consistently passes on `slave-constantine` and fails on `slave-waldorf` and `slave-pepe`
* As far as I know, this is the first time we tried to run GDB on Jenkins nodes (I had to add it to Dockerfile)
* I got shell access to `slave-waldorf` and was able to run GDB manually, and the scenario worked (not inside Docker, though, because I don't have Docker access)
  * specifically, I ran `GDB=1 GDB_SCRIPT=debug.gdb ./pal_loader debug`
* I added some debug prints and GDB fails to even start the program (i.e. `starti` fails), but provides no additional feedback
* No reproduction on any other system (we tried a few)

**Comment by mkow** [comment]
@donporter @dcowhig Please take a look at this, our Jenkins is in quite a bad shape currently (there are also other failures on SGX slaves, I think one is running out of disk space).

**Comment by pwmarcz** [comment]
Fixed during work on #1868.
